cibuildwheel/util.py

```diff
@@ -2,14 +2,14 @@
 import warnings
 
 
-def prepare_command(command, project):
+def prepare_command(command, replace_dict):
```

Comment:
What about `**replace`? Then we can keep the old syntax.

Is there no way to check these format variables that are replaced, though? It would be nice to catch typos rather sooner than later.

Reply:
> What about `**replace`? Then we can keep the old syntax.

Replaced with the standard `**kwargs`.